vue/components/form/inputs/clear_input_button.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+  <button
+    type="button"
+    :style="buttonStyle"
+    @click="clear"
+  >
+    <IonIcon class="field-clear-icon" :style="iconStyle" name="trash-outline"/>
+  </button>
+</template>
+
+<script>
+import IonIcon from 'vue-ionicon';
+
+export default {
+  name: 'DInputClearButton',
+  components: { IonIcon },
+  props: {
+    iconStyle: {
+      type: String,
+      default: 'float: right; transform: none;',
+    },
+    buttonStyle: {
+      type: String,
+      default: '',
+    },
+  },
+  methods: {
+    clear() {
+      this.$emit('clearButtonPressed');
+    },
+  },
+};
+</script>
+
+<style scoped>
+.field-clear-icon {
+  width: 1em;
+  top: 55%;
```

Comment:
In tudi tale je sumljiv. Bo še vedno deloval, če daš width: 2em?

Reply:
odstranjeno sedaj ej stil določen iz parent input elementa